scripts/ci/auto_rebase.sh

```diff
@@ -11,14 +11,14 @@
 
 
 date_str=$(date '+%Y%m%d')
-rebase_branch=features/bot_aicompiler_rebase_${date_str}
-commit_msg="[BOT] aicompiler rebase ${date_str}"
+rebase_branch=features/bladedisc_rebase_${date_str}
+commit_msg="[BOT] bladedisc rebase ${date_str}"
 export tf_commit_body=""
 
 function rebase_tf() {
   set -ex
   base_branch=$(git rev-parse HEAD)
-  git config remote.upstream.url >&- || git remote add upstream git@github.com:tensorflow/tensorflow.git
+  git config remote.upstream.url >&- || git remote add upstream https://github.com/tensorflow/tensorflow.git
```

Comment:
why not using the tls auth ?

Reply:
Its permission was denied. See https://github.com/alibaba/BladeDISC/runs/7831036906?check_suite_focus=true